litgpt/pretrain.py

```diff
@@ -251,11 +252,16 @@ def fit(
                     state["iter_num"] * train.micro_batch_size * model.max_seq_length * fabric.world_size
                 ),
                 "learning_rate": lr,
+                "val_loss": val_loss,
```

Comment:
We can't log the loss here: It is None in the first iteration which is annoying to handle, and we are logging it below after the val loop. 
```suggestion
```